benchmarks/tosi_et_al_2015_gcubed/tosi.cc

```diff
@@ -578,7 +578,7 @@ namespace aspect
       // the values of the compositional fields are stored as blockvectors for each field
       // we have to extract them in this structure
       std::vector<std::vector<double>> prelim_composition_values (this->n_compositional_fields(),
-                                                                  std::vector<double> (n_q_points));
+                                                                   std::vector<double> (n_q_points));
```

Comment:
This is an unfortunate side effect. Because astyle indents with the `>>` in the first line treated as `> >` , it then moves the last line by one space to the right. Fortunately, these occurrences are quite rare, as can be seen from the rest of the patch.

Reply:
Can you rebase again after #4635 is merged? That should get rid of these changes.